help/en/html/scripthelp/DispatcherSystem/DispatcherSystem.shtml

```diff
@@ -1,275 +1,2794 @@
+
+<p class=MsoNormal>A screen will be displayed reminding you that for timetables
+to be displayed the trains must be scheduled and the scheduler be running. If
+these conditions are satisfied you will get a timetable displayed</p>
+
+<p class=MsoNormal>&nbsp;</p>
+
+<p class=MsoNormal>You may thus get a message </p>
+
+<p class=MsoNormal>&nbsp;</p>
+
+<p class=MsoNormal><img border=0 width=337 height=119
+src="DispatcherSystem_files/image133.gif"
+alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>
+
+<p class=MsoNormal>&nbsp;</p>
+
+<p class=MsoNormal>In which case you need to ensure that the clock is running
+and the train scheduler is running by clicking the start scheduler button.</p>
+
+<p class=MsoNormal>&nbsp;</p>
+
+<p class=MsoNormal>A timetable similar to the below timetable will be shown:</p>
+
+<p class=MsoNormal>&nbsp;</p>
+
+<p class=MsoNormal><img border=0 width=883 height=335
+src="DispatcherSystem_files/image134.gif"></p>
+
+<h2><a name="_Toc170509703"></a><a name="_Toc168507399">Remote Timetables</a>
+triggered by MQTT</h2>
+
+<p class=MsoNormal>To display a timetable on a remote tablet or computer
+screen, we need to:</p>
+
+<p class=MsoNormal>&nbsp;</p>
+
+<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'>1)<span
+style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>Set
+up the required MQTT environment using Mosquitto and Node Red on a local or
+remote computer</p>
+
+<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'>2)<span
+style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>Set
+up a local or remote network connection</p>
+
+<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'>3)<span
+style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>Setup
+a MQTT connection in JMRI</p>
+
+<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'>4)<span
+style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>Generate
+the MQTT Node Red Code</p>
+
+<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'>5)<span
+style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>Import
+the code to node red, and perform a trivial edit</p>
+
+<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'>6)<span
+style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>Get
+the appropriate page on a browser to display the timetable</p>
+
+<p class=MsoNormal>&nbsp;</p>
+
+<p class=MsoNormal>These timetables were inspired by a Rocrail MQTT
+implementation and modified from that implementation. The Rocrail &nbsp;implementation
+and hence the JMRI Node Red implementation is licenced as follows<span
+style='font-size:9.0pt;font-family:"Arial",sans-serif;color:#333333;background:
+#FBFAF9'>:&nbsp;</span><a
+href="https://creativecommons.org/licenses/by-sa/4.0/deed.en">CC
+Attribution-Share Alike 4.0 International</a>. Note that this is the only part
```

Comment:
Could we please remove the sentence about "Note that this is the only part of JMRI licensed in this fashion."  Thanks.